app/models/affiliate.rb

```diff
@@ -722,17 +734,13 @@ def set_json_fields
 
   def load_css_properties
     return {} if css_properties.blank?
-    JSON.parse(css_properties, :symbolize_names => true)
+
+    JSON.parse(css_properties, symbolize_names: true)
   end
 
   def clear_existing_attachments
-    if mobile_logo? and !mobile_logo.dirty? and mark_mobile_logo_for_deletion == '1'
-      mobile_logo.clear
-    end
-
-    if header_tagline_logo? and !header_tagline_logo.dirty? and mark_header_tagline_logo_for_deletion == '1'
-      header_tagline_logo.clear
-    end
+    mobile_logo.clear if mobile_logo? && !mobile_logo.dirty? && mark_mobile_logo_for_deletion == '1'
+    header_tagline_logo.clear if header_tagline_logo? && !header_tagline_logo.dirty? && mark_header_tagline_logo_for_deletion == '1'
```

Comment:
Let's revert this change. The original is more readable, and the `Style/IfUnlessModifier` failure for the original is actually a false positive caused by our disabling of the line length cop. The recommendation is to use multiple lines if the condition would be too long for a single line, i.e.
```
if long_condition_that_prevents_code_fit_on_single_line
  do_something_with_a_long_name(arg)
end
```
https://docs.rubocop.org/rubocop/cops_style.html#styleifunlessmodifier

Reply:
Reverted.